docs/embedded/concepts/app-concepts/containertypes.md

```diff
@@ -7,21 +7,23 @@ ms.localizationpriority: high
 
 # SharePoint Embedded Container Types
 
-In SharePoint Embedded, files and documents are stored in Containers, each identified by a Container Type, which is a property stamped on every instance. A Container Type links all Containers to a specific Azure Subscription for billing. A single SharePoint Embedded Application owns each Container Type, and each SharePoint Embedded Application can own only one Container Type. It specifies the access permissions a SharePoint Embedded Application has for all Containers of that type, such as creating, reading, writing, deleting containers, and managing container permissions.
+In SharePoint Embedded, files and documents are stored in Containers, with each container instance stamped with an immutable Container Type property. Each Container Type is owned by only one SharePoint Embedded application, and each SharePoint Embedded application can only own one Container Type. Container Type serves three general purposes:
 
-## Two Kinds of Container Types
+1.	It links all Containers of that type to a specific Azure Subscription for billing. 
```

Comment:
Markdown numbered bullets should always use `1.`... don't use explicit numbers. Let the rendering engine do the numbering for you.